test/remote/gateways/remote_global_collect_test.rb

```diff
@@ -55,6 +55,35 @@ def test_successful_purchase_with_more_options
     assert_equal 'Succeeded', response.message
   end
 
+  def test_successful_purchase_with_airline_data
+    options = @options.merge(
+      airline_data: {
+        code: 111,
+        name: 'Spreedly Airlines',
+        flight_date: '20190810',
+        passenger_name: 'Randi Smith',
+        flight_legs: [
+          { arrival_airport: 'BDL',
+            origin_airport: 'RDU',
+            date: '20190810',
+            carrier_code: 'SA',
+            number: 596,
+            airline_class: 'ZZ'},
+          { arrival_airport: 'RDU',
+            origin_airport: 'BDL',
+            date: '20190817',
+            carrier_code: 'SA',
+            number: 597,
+            airline_class: 'ZZ'}
+        ]
+      }
+    )
+
+    response = @gateway.purchase(@amount, @credit_card, options)
+    assert_success response
+    assert_equal 'Succeeded', response.message
+  end
```

Comment:
Related to my question above, consider adding a test where some of this data is missing, and see that everything is still handled as expected by the adapter as well as the gateway sandbox.